### PR TITLE
Import rpmlintrc

### PR DESCRIPTION
**Pull request by tacerus**
Let's keep this in Git as well.

### CODE REVIEW

File: packaging/infrastructure-formulas-rpmlintrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# yes, libvirt starts with "lib"...
+addFilter('libvirt-formula.noarch: W: shlib-policy-missing-lib')
+addFilter('infrastructure-formulas.noarch: W: explicit-lib-dependency libvirt-formula')
+
+# this is a meta-package installing all the formula subpackages, it doesn't need any files
+addFilter('infrastructure-formulas.noarch: W: suse-filelist-empty packages without any files are discouraged in SUSE')
+
+# no it is not
+addFilter('\w+-formula.noarch: W: description-shorter-than-summary')
```
Comment (cboltz) [inline packaging/infrastructure-formulas-rpmlintrc:9]: Looks like dots don't count in rpmlint ;-) - maybe open a bugreport for it?

That said: I'd prefer not to have this filter in the rpmlintrc - or at least it should be limited to subpackages affected by the rpmlint bug.

Besides that, this PR looks good.

Comment (tacerus) [reply packaging/infrastructure-formulas-rpmlintrc:9]: :-) I might.

It should already be limited to the subpackages, but it cannot quite be limited to specific ones without listing each one individually. So, rather drop it?

Comment (tacerus) [reply packaging/infrastructure-formulas-rpmlintrc:9]: I think a bug report is not needed, the warning does not appear with the rpmlint version in Factory.

Comment (cboltz) [reply packaging/infrastructure-formulas-rpmlintrc:9]: Sounds like dropping it (and reading only the rpmlint report for Tumbleweed) would be a good idea.

If you don't want to drop it, I'd indeed list each subpackage individually, with a comment that these lines are only needed for Leap 15.x.

Comment (tacerus) [reply packaging/infrastructure-formulas-rpmlintrc]: Fair, I took the easy way.

**Pull request merged**